packages/aws-client/src/S3.ts

```diff
@@ -1001,7 +1003,7 @@ export const multipartCopyObject = async (
       throw new Error(`Unable to determine size of s3://${sourceBucket}/${sourceKey}`);
     }
 
-    const chunks = S3MultipartUploads.createMultipartChunks(objectSize);
+    const chunks = S3MultipartUploads.createMultipartChunks(objectSize, maxChunkSize);
```

Comment:
I wonder if we should check that we haven't gone over the maximum number of chunks?  "A multipart upload requires that a single file is uploaded in not more than 10,000 distinct parts. You must be sure that the chunksize that you set balances the part file size and the number of parts."   If that array is longer than 10000 we could throw an error?

Reply:
Since the minimal chunk size is 5MB, I tried to create 55,000MB size file for testing, and got 502 error from localstack, and I think 50GB file is too big for testing.

Reply:
yup I didn't do that math.

Reply:
And if the default value is 250MBs, that would be a 2.5TB file before it was broken.  I wonder if AWS would have a good enough error message for us to know we broke the limits

Reply:
My concern is that aws limit could change, and we don't want to hardcode the limit.

Reply:
good point. I think this is good to go. I'll approve